client/src/components/SolutionViewer/ProjectModal.tsx

```diff
@@ -15,20 +16,19 @@ const propTypes = {
   //     path: PropTypes.string
   //   })
   // ),
-  handleSolutionModalHide: PropTypes.func,
-  isOpen: PropTypes.bool,
-  projectTitle: PropTypes.string,
-  solution: PropTypes.string
+  handleSolutionModalHide: () => void;
+  isOpen: boolean;
+  projectTitle: string;
+  solution: string | null | undefined;
```

Comment:
```suggestion
  solution: string;
```

Reply:
This throws a compiler error in the `show-project-links.tsx` file because the initial state of `solution` is `null`.

Reply:
S'why I proposed the `String(solution)` approach.

hacky, and I'm open to a better way to do it, but it would prevent blocking this PR.